tutorials/environmental_data_and_visualization.md

```diff
@@ -0,0 +1,173 @@
+\page environmental_data Visualizing Environment Data
+
+Lot of times you may come accross environment properties that vary with time and
+space. For instance if you'd like to model properties such as gas distribution
+and water temperature one needs to have some way of defining values such that
+the values vary through out the environment. Gazebo's Environment Data
+infrastructure enables such visualization and sensing capabilities.
+
+# Loading Environment Data
+
+Currently gazebo is able to load data from CSV files and display them. It is
+assumed the points are sampled at a fixed grid interval. For gaps in between the
+sample points, Gazebo will interpolate the data.
+
+## Data Format
+
+As mentioned earlier, the data should be a CSV file with grid based spacing.
+The first line of the CSV file should be the heading for each column.
+There should be a column for time (in seconds), the x coordinate, y coordinate
+and z coordinate (in meters). You may have multiple columns of data like so:
```

Comment:
@arjo129 nit: it may be worth listing/explaining the reference frames in which these spatial coordinates can be defined.